tests/packagedcode/data/phpcomposer/a-timer/composer.json.expected

```diff
@@ -0,0 +1,82 @@
+{
+  "type": "phpcomposer", 
+  "name": "jandreasn/a-timer", 
```

Comment:
Is there some meaning in the composer world to have xxxx/yyyyy ? do xxx and yyy mean something? or the whole is the package name?
Is this related to github's onwer/repo somehow?